doc/api/dgram.md

```diff
@@ -552,17 +552,16 @@ chained.
 ### Change to asynchronous `socket.bind()` behavior
 
 As of Node.js v0.10, [`dgram.Socket#bind()`][] changed to an asynchronous
-execution model. Legacy code that assumes synchronous behavior, as in the
-following example:
+execution model. Legacy code would use synchronous behavior:
 
 ```js
 const s = dgram.createSocket('udp4');
```

Comment:
Super tiny Nit: If we say, this is some code which used to work in older versions, I think it would be better to change this `const` to `var`.

Reply:
It seems some latest versions before the change support `const`:
```js
> process.versions
{ http_parser: '1.0',
  node: '0.8.28',
  v8: '3.11.10.26',
  ares: '1.7.5-DEV',
  uv: '0.8',
  zlib: '1.2.3',
  openssl: '1.0.0m' }
> const a = 1;
undefined
> a
1
```

Reply:
But I can change (with linter disabling comment) if it makes the difference more obvious.